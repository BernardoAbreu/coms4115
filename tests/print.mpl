tuple Test {a, b, c:integer}

t = Test;
t2 = Test;

t$a = "ta";
t$b = "tb";
t$c = 123;

t2$a = "t2a";
t2$b = "t2b";
t2$c = 456;

print(7);
print("\n");
print(2.67);
print("\n");
print("abc");
print("\n");
print(t);
print("\n");

arr_int = [1, 2, 3, 4, 5];
arr_float = [8.12, 9.123, 64.23, 3.1415];
arr_str = ["hello", "bonjour", "ola"];
arr_tup = [t2, t];

print(arr_int);
print(arr_float);
print(arr_str);
print(arr_tup);

arr_str2 = ["world", "monde", "mundo"];

arr_arr = [arr_str, arr_str2];

print(arr_arr);