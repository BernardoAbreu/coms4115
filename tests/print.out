7
2.67
abc
(ta, tb, 123)
[ 1, 2, 3, 4, 5 ]
[ 8.12, 9.123, 64.23, 3.1415 ]
[ hello, bonjour, ola ]
[ (t2a, t2b, 456), (ta, tb, 123) ]
[ [ hello, bonjour, ola ][ world, monde, mundo ] ]
